test/integration/next-image-legacy/typescript/test/index.test.ts

```diff
@@ -75,16 +75,18 @@ describe('TypeScript Image Component', () => {
     })
   })
 
-  it('should remove global image types when disabled (dev)', async () => {
-    const content = await fs.readFile(nextConfig, 'utf8')
-    await fs.writeFile(
-      nextConfig,
-      content.replace('// disableStaticImages', 'disableStaticImages')
-    )
-    const app = await launchApp(appDir, await findPort())
-    await killApp(app)
-    await fs.writeFile(nextConfig, content)
-    const envTypes = await fs.readFile(join(appDir, 'next-env.d.ts'), 'utf8')
-    expect(envTypes).not.toContain('image-types/global')
+  describe('development mode 2', () => {
```

Comment:
Is there a reason this test was moved into a new `describe`? Is it related to concurrency or something?

Reply:
Oh maybe because it doesn't need beforeAll/afterAll?